Explorer/Assets/DCL/Interaction/Systems/PlayerOriginatedRaycastSystem.cs

```diff
@@ -53,12 +53,12 @@ protected override void Update(float t)
         }
 
         [Query]
-        private void RaycastFromCamera(ref CameraComponent camera, in CursorComponent cursorComponent)
+        private void RaycastFromCamera(Entity entity, ref CameraComponent camera, in CursorComponent cursorComponent)
         {
             ref PlayerOriginRaycastResultForSceneEntities raycastResultForSceneEntities = ref playerInteractionEntity.PlayerOriginRaycastResultForSceneEntities;
             ref PlayerOriginRaycastResultForGlobalEntities raycastResultForGlobalEntities = ref playerInteractionEntity.PlayerOriginRaycastResultForGlobalEntities;
 
-            if (cursorComponent.CursorState == CursorState.Panning)
+            if (cursorComponent.CursorState == CursorState.Panning || World.Has<DCL.InWorldCamera.ScreencaptureCamera.InWorldCamera>(entity))
```

Comment:
What does the raycastResultForSceneEntities resetting do? Why having an in-world camera requires resetting them?

Reply:
it is for example to highlight interactable entity over cursor. In that case of camera mode - everything should be not interactable. So the one problem was if it was already highlighted when toggling into in-world camera mode, then highlight remained. This reseting functionality helps to get rid of it (outline, player name or other highlighting feature)